source/common/router/header_formatter.cc

```diff
@@ -336,11 +337,11 @@ StreamInfoHeaderFormatter::StreamInfoHeaderFormatter(absl::string_view field_nam
       const auto& formatters = start_time_formatters_.at(pattern);
       std::string formatted;
       for (const auto& formatter : formatters) {
-        absl::StrAppend(&formatted,
-                        formatter->format(*Http::StaticEmptyHeaders::get().request_headers,
-                                          *Http::StaticEmptyHeaders::get().response_headers,
-                                          *Http::StaticEmptyHeaders::get().response_trailers,
-                                          stream_info, absl::string_view()));
+        const auto& bit = formatter->format(*Http::StaticEmptyHeaders::get().request_headers,
```

Comment:
Seems like the result of format is not a reference, so I think the `&` is incorrect since the result is temporary.